wasm-rpc-stubgen/src/model/app.rs

```diff
@@ -272,18 +299,27 @@ impl<CPE: ComponentPropertiesExtensions> Application<CPE> {
                 validation.push_context("component", component_name.to_string());
 
                 for dependency in component_dependencies {
-                    if dependency.type_ == "wasm-rpc" {
+                    if dependency.type_ == "static-wasm-rpc" || dependency.type_ == "wasm-rpc" {
```

Comment:
now that we have multiple values maybe we could add these as consts to app_raw (or we could make it an enum there, but this way error reporting is more lenient)

Reply:
ok